docs/modules/pipelines/pages/job-management.adoc

```diff
@@ -184,12 +184,13 @@ ALTER JOB hello-world RESUME;
 
 You can also update the configuration of a suspended job and resume it.
 
+For memory management, the Jet engine limits the number of objects individual processors store. If this number is exceeded the job fails. To resolve this type of error, you can update the job configuration to process fewer items, or add more resources. The following example shows how to decrease the number of records that each processor can store.
```

Comment:
> ..., you can update the job configuration to process fewer items, ...

You cannot process less items by updating the job configuration since it is tied to the job definition. Currently, this can only be done by cancelling the job and submitting another. The original paragraph is the following and I think it can be used with little change (the error is now recoverable thanks to `Job.updateConfig()`).

> Currently, Jet processors implement only a very basic memory management by limiting the number of objects individual processors store. In case the number is exceeded, the job fails. This error is currently unrecoverable, because the user can’t fix it e.g. by processing less items, or by adding more resources. One strategy can be to send excess items to the error sink. Another is if we allow increasing this limit at runtime and resume.